sys/console/syscfg.yml

```diff
@@ -20,7 +20,7 @@ syscfg.defs:
     CONSOLE_IMPLEMENTATION:
         description: >
             Selects console implementation package full, minimal or stub.
-        value:
+        value: stub
```

Comment:
I'd go for 'full' as it may be a bit counter-intuitive to have non-functional console by default